community/becoming-a-maintainer.md

```diff
@@ -1,3 +1,77 @@
 # Becoming a maintainer
 
-TODO
+## Introduction
+
+Our Maintainers are the people within Exercism that manage the different parts of our ecosystem. 
+We aim to have at least active two maintainers for each programming language track, and at least one active maintainer for all the parts of the platform.
+
+We have a formalised process for becoming and staying a maintainer, to make it achievable and fair for anyone.
+**You can be become a maintainer!**
+Read on to understand the responsibilities and the process.
+
+_Note: the sections below generally discuss Track Maintainers. 
+The process for becoming a maintainer of a different part of the ecosystem is similar but a little looser._
+
+## Responsibilities
+
+Maintainers are responsible for ensuring that a Track works well within Exercism, and continues to grow and develop. 
+That specifically means that maintainers:
+- **Set technical direction for a track:** Choosing the right language versions, tooling, etc.
+- **Manage the syllabus of a track:** Designing, choosing and defining the right Concepts and Exercises for a Track, and ensuring they are linked and structured appropriately.
+- **Ensure a traack track work well:** Ensuring the Track is up to date, that its test-runner is fast, etc.
+- **Ensure the track conforms to Exercism's standard:** Exercism has specifications and standards for each language Track to ensure a degree of consistency within the platform. While maintainers have free reign over much of the Track, all Tracks must conform to Exercism's standards.
+
+Maintainers are also responsibile for interacting with our contributor community.
+They review PRs from community, acting as gatekeepers, shephards, and cheerleaders for our community.
+They are responsible for keeping documentation and contribution guidelines up to date, and ensuring that there are relevant open issues for potential contributors to action.
+
+It is essential that Maintainers manifest Exercism's values of empathy and kindness, while maintaining high standards for the Track and the platform.
+
+## Powers
+
+The primary way that Maintainers are able to enact their responsibilities is by having the write access to repositories. 
```

Comment:
```suggestion
The primary way that Maintainers are able to enact their responsibilities is by having the right access to repositories. 
```

Reply:
This means "write access" (as in being allowed to write to a repo, or the write "bit")